src/registry/addressNL/edit.tsx

```diff
@@ -45,6 +46,26 @@ const DeriveAddress = () => {
   );
 };
 
+const ColumnsLayout: React.FC = () => (
+  <>
+    <Select
+      name="layout.show"
+      label={
+        <FormattedMessage
+          description="Component property 'layout.show' label"
+          defaultMessage="Choose whether to render the component in a single or double column"
```

Comment:
```suggestion
          defaultMessage="Layout"
```
keep labels simple :)